chart/templates/controller.yaml

```diff
@@ -0,0 +1,202 @@
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otp-controller-manager

```

Comment:
if we had different transport technology, would something be different in the service account/roles?
I think not.
I would call the things `transport-controller..` instead of `otp..`. there is nothing here that is otp specific.

in the future if we have more than one option for transport, I envision the common part to be created as part of the KubeStellar repo chart, and the plugin repo to create only its specifics.
in that case, I envision the service account + role + rolebinding of transport to be created in KS chart.

for now since we have only one option, it seems better to put it here. 
but as a preparation for future, let's just call the objects `transport-controller` without `otp` unless it's specific to otp.

Reply:
ok, changed to `transport-controller-sa`